crypto/random/random.go

```diff
@@ -0,0 +1,45 @@
+package random
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/binary"
+
+	"github.com/consensys/gnark-crypto/ecc/bls12-381/fr"
+	GoKZG "github.com/crate-crypto/go-kzg-4844"
+	"github.com/sirupsen/logrus"
+)
+
+// deterministicRandomness creates a deterministic 32 byte array from a seed
```

Comment:
```suggestion
// DeterministicRandomness creates a deterministic 32 byte array from a seed
```

godoc should start with the exact function name (case)